src/UniswapV3Swapper.sol

```diff
@@ -31,14 +31,16 @@ contract UniswapV3Swapper is ISwapper, Ownable {
     /// @notice Thrown when the zero address is passed as input.
     error ZeroAddress();
 
-    /// STORAGE ///
+    /// CONSTANTS AND IMMUTABLES ///
```

Comment:
We could separate constants and immutables to make it even clearer

Reply:
Honestly this is getting heavier & heavier... Comments are supposed to add clarity to the code, but here it's useless to me: the reader definitely knows that the following block contains constants & immutables because they are grouped together (and the developer knows too for the same reason)
My POV is similar for the constructor, but I do find it useful for separating external & internal functions because there are plenty of these

The issue with this is that it is adding friction to PRs: we never agree on the same comments, we argue over whether we should write `&` or `AND` or `APPENDIX` inside separators whereas it has zero impact on audit, clarity of code or maintainability to me

Reply:
I think the constants should not be separated from immutables, as @Rubilmax says it does add some pointless friction to the PRs. I don't think there's a good reason to separate the two. I do think separating storage makes sense because this actually provides clarity as to the storage ordering of the contract.

Reply:
Agree with you let's keep it like that!